controllers/scriptStorage.js

```diff
@@ -156,7 +162,9 @@ exports.sendScript = function (aReq, aRes, aNext) {
     ++aScript.installs;
     ++aScript.installsSinceUpdate;
 
-    aScript.save(function (aErr, aScript) { });
+    aScript.save(function (aErr, aScript) {
+      // WARNING: Resaving the script... WHY?
```

Comment:
@sizzlemctwizzle Cc: @Zren
Is there a particular reason why we are resaving the `Script` object here?

Seems to me this may be a logic issue if it is truly needed as the function name implies that it is just `sendScript`... possible migration artifact as well. e.g. dead code and/or undocumented migration point.